packages/jest-config/src/Defaults.ts

```diff
@@ -38,6 +38,7 @@ const defaultOptions: Config.DefaultOptions = {
   moduleFileExtensions: ['js', 'json', 'jsx', 'ts', 'tsx', 'node'],
   moduleNameMapper: {},
   modulePathIgnorePatterns: [],
+  moduleWrapper: true,
```

Comment:
If we can avoid config and feature-detect, let's do it please

Reply:
main reason I added it was so people could opt out if it was buggy. It should be safe to just enable though, as long as all of our own tests still pass